README.md

```diff
@@ -42,28 +42,49 @@ a default build, you will additionally need libxml2 and libsqlite3.
 
 On Ubuntu, you can install these using:
 
-    sudo apt install -y pkg-config build-essential autoconf bison re2c \
-                        libxml2-dev libsqlite3-dev
+```shell
+sudo apt install -y pkg-config build-essential autoconf bison re2c libxml2-dev libsqlite3-dev
+```
+
+On MacOS, you can install these using:
+
+```shell
+brew install autoconf bison re2c iconv libxml2 sqlite
```

Comment:
I wouldn't assume Homebrew; I'm using MacPorts instead and most instructions I see that mention brew packages will also mention the MacPorts equivalent.

Reply:
If you know the equivalent for MacPort instruction I'll add it

Reply:
That should resemble something like this

```shell
sudo port install autoconf bison re2c libiconv libxml2 sqlite3
```